pkgs/top-level/all-packages.nix

```diff
@@ -12468,7 +12468,9 @@ let
 
   sxhkd = callPackage ../applications/window-managers/sxhkd { };
 
-  msmtp = callPackage ../applications/networking/msmtp { };
+  msmtp = callPackage ../applications/networking/msmtp {
+    inherit (darwin.apple_sdk.frameworks) Security;
```

Comment:
I've just tried it on my Debian Linux box. As I've assumed, it does not fail on non-Apple. I've verified that it works on Linux.